RELEASENOTES.md

```diff
@@ -8,6 +8,30 @@
 - Issue 1627 AL-Go should throw an error or a warning if you create a release, which is older than the latest release
 - Issue 1657 When no files modified on Git, deployment fails
 
+### New Workflow specific settings
+
+- `buildEnvironmentName` - should be the Business Central environment name (from the Admin Center) for the environment you want to use if you want to run tests using online environments. Note that you also need to create a [buildAuthContext](https://aka.ms/algosecrets#buildAuthContext) secret, which gives access to this environment.
+
+### Use online environment during build for running tests
+
+By creating a [build authentication context](https://aka.ms/algosecrets#buildAuthContext) secret and setting the [buildEnvironmentName](https://aka.ms/algosettings#buildEnvironmentName) for a specific branch and a specific workflow, will cause AL-Go for GitHub to use this environment for running tests. An example of how to define these settings could be:
+
+```json
+  "useCompilerFolder": true,
+  "doNotPublishApps": true,
```

Comment:
Would it make sense to recommend using the workflowConcurrency setting in this example as well? I'm guessing it would fail if you have two CI/CDs running concurrently.